src/EnergyPlus/ScheduleManager.cc

```diff
@@ -313,6 +320,10 @@ namespace Sched {
         auto *schedOn = AddScheduleConstant(state, "Constant-1.0", 1.0);
         assert(schedOn->Num == SchedNum_AlwaysOn);
         schedOn->isUsed = true; // Suppress unused warnings
+
+        auto *missingDaySchedule = AddDaySchedule(state, "MissingDaySchedule-0.0");
+        assert(missingDaySchedule->Num == SchedNum_AlwaysOff);
+        missingDaySchedule->isUsed = true;
```

Comment:
InitConstantScheduleData: Add a missingDaySchedule at index 0 in dataSched->daySchedules